MinimedKit/PumpEvents/DailyTotal523PumpEvent.swift

```diff
@@ -33,7 +33,7 @@ public struct Sara6EPumpEvent: PumpEvent {
     
     public var dictionaryRepresentation: [String: AnyObject] {
         return [
-            "_type": "Sara6E",
+            "_type": "DailyTotal523",
```

Comment:
Nice, though I'll always remember this as Sara6e. I want to get around decoding this event, since this sort of daily-summary data is what I'd like to report on long-term. I was thinking of setting Loop to download history once daily even if the preference is set to Reservoir, just for archive/analysis.

Do you have a recent carelink pull you wouldn't mind contributing to that effort, possibly for the unit test case below?